Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM alpine:3.9
 MAINTAINER Werner Beroux <werner@beroux.com>
 
+RUN apk add --update coreutils && rm -rf /var/cache/apk/*
```

Comment:
Use `apk add --no-cache coreutils`. It does the `--update` and avoids `rm -rf ...`